boards/AVR_MKS_GEN_L_V2/pins_MKS_GEN_L_V2.h

```diff
@@ -236,6 +236,9 @@
 #endif
 
 //Pin to turn on dew heater MOSFET
-#ifndef DEW_HEATER_PIN
-    #define DEW_HEATER_PIN 10
+#ifndef DEW_HEATER_PIN1
```

Comment:
Should we promote DEW_HEATER_PIN to DEW_HEATER_1_PIN? For backwards compat...

Reply:
These names are only used within b_setup and the pins files. If someone has a custom pins file for a different atmega2560 board other than the MKS **and** they're using dew heaters, they'd have to update their pins, but that would be very unlikely.